docs/xgc.md

```diff
@@ -35,7 +35,9 @@ Options that change the behavior of the Garbage Collector (GC).
 | Parameter                                                                 | Effect                                                                                                    |
 |---------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------|
 | [`breadthFirstScanOrdering`         ](#breadthfirstscanordering         ) | Sets the scan mode to breadth first.                                             |
-| [`concurrentScavenge`               ](#concurrentscavenge               ) | Enables a garbage collection (GC) mode with less pause times.                                             |
+| [`classUnloadingKickoffThreshold`   ](#classunloadingkickoffthreshold   ) | Sets a threshold to start an early concurrent global garbage collection (GC) cycle due to class unloading activity  |
```

Comment:
It's 'cycle due to class _loading_ activity' or we can even emphasize it's _recent_,  _heavy_ (_more that usual_).

Some more background, not necessarily that should go into the doc...
Because there was a lot of recent class loading activity by application, GC decided to trigger early global to unload some classes that potentially died. And all that to reduce amount of class unloading per GC cycle, to reduce pauses due to class unloading.